src/udev/udevd.c

```diff
@@ -770,6 +778,14 @@ static int is_device_busy(Manager *manager, struct event *event) {
         return false;
 
 set_delaying_seqnum:
+        if (DEBUG_LOGGING) {
+                uint64_t seqnum;
+
+                r = device_get_seqnum(event->dev, &seqnum);
+                log_device_debug(event->dev, "SEQNUM=%" PRIu64 " blocked by SEQNUM=%" PRIu64,
+                                 r >= 0 ? seqnum : 0, loop_event->seqnum);
```

Comment:
Or maybe, `event->seqnum` can be used here? If so, we can omit `device_get_seqnum()`.

Reply:
I thought I had this at some point and resulted in a compile error. Fixed.